demo_serial/README.md

```diff
@@ -7,5 +7,5 @@ This is a copy of the http://developer.android.com/reference/android/app/Downloa
 
 Here we show two use cases
 
-   - Downloading from a url
+   - Downloading items **serially** from a url
```

Comment:
`- Downloading items **serially** from a url (this means the second will not start till the first completes)`